mlops/parallelm/mlops/mlops_metrics.py

```diff
@@ -166,6 +166,35 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
 
         return bsl
 
+    @staticmethod
+    def classification_report(y_true, y_pred,
+                              labels=None, target_names=None,
+                              sample_weight=None,
+                              digits=2):
+        """
+        Method generates classification report and output it using MCenter as table.
```

Comment:
MCenter is not really relevant here..

Reply:
yes it is not really relevant. but method will be accessed mostly by channels that mcenter provides!